core/genesis.go

```diff
@@ -299,6 +302,13 @@ func (g *Genesis) Commit(db ethdb.Database) (*types.Block, error) {
 		return nil, err
 	}
 
+	genesisJSON, err := g.MarshalJSON()
+	err = db.Put(DBGenesisKey, genesisJSON)
```

Comment:
If there are multiple accounts in the genesis json file, wouldn't this be overwritten by all but one of those accounts?

Reply:
Why? This just stores the entire genesis object

Reply:
Yep.  Your right.  I misread the code.